### PR TITLE
light refactor

### PR DESCRIPTION
**Pull request by AlexanderHHansen**

### CODE REVIEW

**Review by frejaz — APPROVED** [review]
Looks finno

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Install Python dependencies
-        run: pip install -r ./scripts/requirements.txt
```
Comment (CasperKjaerhus) [inline .github/workflows/pytest.yml:25]: Kan du tilføje det her til docker containeren? :)

Comment (CasperKjaerhus) [reply .github/workflows/pytest.yml:25]: Altså at requirements filen er en del af containeren

File: geo_json_metrics/gps_metrics.py [hunk 2]
```diff
@@ -228,20 +228,29 @@ def calculate_metrics(df: pd.DataFrame) -> tuple[float, float, float]:
     return (median_avg, median_min, median_max)
 
 
+def verify_outlier(filtered_df: pd.DataFrame):
+    """Used to verify the outlier from segment_10240935_linestring.json.
+    The speed calculated was initially wrong. But now fixed.
```
Comment (JakobFaarGreg) [inline geo_json_metrics/gps_metrics.py:233]: Can this be made more generic? It seems out-of-nowhere that specifically this line string has a specific outlier.

File: geo_json_metrics/gps_metrics.py [hunk 3]
```diff
@@ -228,20 +228,29 @@ def calculate_metrics(df: pd.DataFrame) -> tuple[float, float, float]:
...
+
+    Args:
+        filtered_df (pd.DataFrame): dataframe
+    """
+    print(filtered_df.iloc[7].time_difference)
```
Comment (JakobFaarGreg) [inline geo_json_metrics/gps_metrics.py:238]: delete?

File: geo_json_metrics/gps_metrics.py [hunk 4]
```diff
@@ -228,20 +228,29 @@ def calculate_metrics(df: pd.DataFrame) -> tuple[float, float, float]:
...
+
+    for (x, y, time), speed in zip(
+        filtered_df.iloc[7]["coordinates"], filtered_df.iloc[7]["speeds"]
+    ):
+        print(time, speed, sep=", ")
```
Comment (JakobFaarGreg) [inline geo_json_metrics/gps_metrics.py:243]: delete?

**Review by JakobFaarGreg — CHANGES_REQUESTED** [review]
Jeg har få kommentarer

File: geo_json_metrics/gps_metrics.py (hunk 2 above)
Comment (AlexanderHHansen) [reply geo_json_metrics/gps_metrics.py:233]: done

File: geo_json_metrics/gps_metrics.py (hunk 4 above)
Comment (AlexanderHHansen) [reply geo_json_metrics/gps_metrics.py:243]: keep

File: geo_json_metrics/gps_metrics.py (hunk 3 above)
Comment (AlexanderHHansen) [reply geo_json_metrics/gps_metrics.py:238]: deleted

**Review by JakobFaarGreg — APPROVED** [review]
lgtm

**Review by JakobFaarGreg — APPROVED** [review]

**Review by JakobFaarGreg — APPROVED** [review]

**Pull request merged**